### PR TITLE
Skeleton Application Docs

### PR DESCRIPTION
**Pull request by fntlnz**

### CODE REVIEW

File: docs/skeleton-application.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+## Get it
+
+The Penny skeleton application is [hosted on GitHub](https://github.com/gianarb/penny-classic-app)
+
+The most common ways to get it are: 
+
+### Via git clone
+
+```bash
+https://github.com/gianarb/penny-classic-app.git
```
Comment (samsonasik) [inline docs/skeleton-application.md]: git clone https://github.com/gianarb/penny-classic-app.git

File: docs/skeleton-application.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+```
+wget -nv -O - https://github.com/gianarb/penny-classic-app/archive/master.zip | tar zx
+```
+
+## Requirements
+
+- PHP >= 5.4
+- Composer (Required to manage PHP dependencies)
+- Node and npm  (Required to build frontend assets)
```
Comment (samsonasik) [inline docs/skeleton-application.md]: add fullstop (`.`) at each list or at the end of list.

File: docs/skeleton-application.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+## Build assets
+
+Javascript frontend dependencies are managed trough [bower](http://bower.io/) and built using [grunt](http://gruntjs.com).
+Grunt and other build tools are 
+
+**Note:** *The following commands must be issued in the skeleton application folder*
+
+*Resolve node dependencies*
+
+```
```
Comment (samsonasik) [inline docs/skeleton-application.md]: add ```bash

File: docs/skeleton-application.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+sudo npm install -G grunt-cli
+npm install
+```
+
+*Resolve frontend dependencies*
+
+```
```
Comment (samsonasik) [inline docs/skeleton-application.md]: add ```bash

**Comment by fntlnz** [comment]
@samsonasik Thanks for your review !

**Comment by gianarb** [comment]
Thanks!!

**Pull request merged**